CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+### citus v5.2.0 (August 15, 2016) ###
+
+* Drops support for PostgreSQL 9.4; PostgreSQL 9.5 is required
+
+* Adds support for schema use in queries and functions
+
+* Evaluates non-immutable functions on master in INSERT and UPDATE commands
+
+* Adds support for SERIAL types in non-partition columns
+
+* Adds support for RETURNING clause in INSERT, UPDATE, and DELETE commands
+
+* Adds support for multi-statement transactions involving a fixed set of nodes
+
+* Adds option to perform DDL changes using prepared transactions (2PC)
+
+* Adds an `enable_ddl_propagation` parameter to control DDL propagation
+
+* Now detects single-shard queries and automatically uses router executor
+
+* Accelerates shard pruning during merges
+
+* Adds `master_modify_multiple_shards` UDF to modify to many shards at once
+
+* Adds COPY support for arrays of user-defind types
+
+* Now supports parameterized prepared statements for certain use cases
+
+* Greatly improves router-plannable query SQL coverage
```

Comment:
Merged these two lines into _Full SQL support for SELECT queries which can be executed on a single machine_. Tell me if that's too extreme, though I think it's accurate?